projects/demo/src/modules/info/testing/disable-animation/disable-animation.component.ts

```diff
@@ -3,8 +3,10 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
 @Component({
     selector: 'disable-animation',
     templateUrl: './disable-animation.template.html',
+    styleUrls: ['../../../../modules/app/home/home.style.less'],
```

Comment:
I think we should either copy-paste or create a component with those styles, I don't like the idea of entanglement of 2 separate pages with the styles.

Reply:
fixed